src/client/pages/Varen/Varen.tsx

```diff
@@ -0,0 +1,170 @@
+import { useMemo } from 'react';
+
+import styles from '../Vergunningen/Vergunningen.module.scss';
+import { AppRoutes } from '../../../universal/config/routes';
+import { isError, isLoading } from '../../../universal/helpers/api';
+import { defaultDateFormat } from '../../../universal/helpers/date';
+import { getCustomTitleForVergunningWithLicensePlates } from '../../../universal/helpers/vergunningen';
+import { CaseType } from '../../../universal/types/vergunningen';
+import {
+  ErrorAlert,
+  Linkd,
+  MaintenanceNotifications,
+  PageContent,
+  PageHeading,
+  SectionCollapsible,
+  Table,
+  ThemaIcon,
+  addTitleLinkComponent,
+} from '../../components';
+import { OverviewPage } from '../../components/Page/Page';
+import { ThemaTitles } from '../../config/thema';
+import { useAppStateGetter } from '../../hooks/useAppState';
+import { hasMultiplePermits } from '../VergunningDetail/WVOS';
+import { Vergunning } from '../../../server/services';
+
+export const DISPLAY_PROPS = {
+  identifier: 'Kenmerk',
+  title: 'Soort vaar',
+  dateRequest: 'Aangevraagd',
+};
+
+export const DISPLAY_PROPS_HISTORY = {
+  identifier: 'Kenmerk',
+  title: 'Soort vaar',
+  decision: 'Resultaat',
+};
+
+const titleTransformMap: Record<string, any> = {
+  [CaseType.TouringcarJaarontheffing]:
+    getCustomTitleForVergunningWithLicensePlates,
+  [CaseType.TouringcarDagontheffing]:
+    getCustomTitleForVergunningWithLicensePlates,
+  [CaseType.EigenParkeerplaats]: getCustomTitleForVergunningWithLicensePlates,
+  [CaseType.EigenParkeerplaatsOpheffen]:
+    getCustomTitleForVergunningWithLicensePlates,
+};
+
+export default function Varen() {
+  const { VAREN } = useAppStateGetter();
+
+  const Varen: Vergunning[] = useMemo(() => {
+    if (!VAREN.content?.length) {
+      return [];
+    }
+    const items: Vergunning[] = VAREN.content
+      .filter((x) => x)
+      .map((item) => {
+        return {
+          ...item,
+          title:
+            item.caseType in titleTransformMap
+              ? titleTransformMap[item.caseType](item)
+              : item.title,
+          dateRequest: defaultDateFormat(item.dateRequest),
+        };
+      });
+    return addTitleLinkComponent(items, 'identifier');
+  }, [VAREN.content]);
+
+  // const varenPrevious = useMemo(() => {
+  //   return varen
+  //     .filter((vaar) => vaar.processed)
+  //     .map((vaar) => {
+  //       if (
+  //         vaar.caseType === CaseType.WVOS &&
+  //         hasMultiplePermits(vaar)
+  //       ) {
+  //         return {
+  //           ...vaar,
+  //           decision: 'Zie besluit',
+  //         };
+  //       }
+  //       return vaar;
+  //     });
+  // }, [varen]);
+
+  // const varenActual = useMemo(() => {
+  //   return varen.filter((vaar) => !vaar.processed);
+  // }, [varen]);
+
+  // const hasActualGPK = varenActual.find(
+  //   (vaar) => vaar.caseType === CaseType.GPK
+  // );
+
+  return (
```

Comment:
Graag het nieuwe ThemaPagina component gebruiken, zie ook `HLI.tsx`

Reply:
Deze Varen.tsx is totaal nog niet aan gewerkt ik heb deze alleen even aangemaakt omdat ik dacht dat ik daarmee het Thema getoond kreeg > volgens mij moet dit in de volgende story Frontend Varen aanmaken worden opgepikt. Ik heb deze file opnieuw gepushed met een comment erin dat dit "fake' data is..

Reply:
Je kunt dan ook iets doen als:

```
export default function Varen() {
 return 'FAKE';
}
```